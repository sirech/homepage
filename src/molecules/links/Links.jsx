import React from 'react'

import { Card, CardTitle, CardBody, ListGroup, ListGroupItem } from 'reactstrap'

const Links = () => (
  <Card className='mb-3'>
    <CardBody>
      <CardTitle tag='h5' className='mb-0'>Links</CardTitle>
    </CardBody>
    <ListGroup flush>
      <ListGroupItem>
        <a href='https://github.com/sirech'>
          Github Repositories
        </a>
      </ListGroupItem>
      <ListGroupItem>
        <a href='https://stackoverflow.com/users/3785/mario-f'>
          Stackoverflow
        </a>
      </ListGroupItem>
    </ListGroup>
  </Card>
)

export default Links
