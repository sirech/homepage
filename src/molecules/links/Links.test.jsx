import React from 'react'
import renderer from 'react-test-renderer'
import Links from './Links'

describe('components', () => {
  describe('Links', () => {
    it('renders correctly', () => {
      const component = renderer.create(<Links />)
      expect(component.toJSON()).toMatchSnapshot()
    })
  })
})
