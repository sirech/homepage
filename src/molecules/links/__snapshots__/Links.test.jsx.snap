// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Links renders correctly 1`] = `
<div
  className="mb-3 card"
>
  <div
    className="card-body"
  >
    <h5
      className="mb-0 card-title"
    >
      Links
    </h5>
  </div>
  <ul
    className="list-group list-group-flush"
  >
    <li
      className="list-group-item"
    >
      <a
        href="https://github.com/sirech"
      >
        Github Repositories
      </a>
    </li>
    <li
      className="list-group-item"
    >
      <a
        href="https://stackoverflow.com/users/3785/mario-f"
      >
        Stackoverflow
      </a>
    </li>
  </ul>
</div>
`;
